From f91bc2e2b4679d73479545feae464e7f6c6ac2cf Mon Sep 17 00:00:00 2001
From: Joris Vaillant <joris.vaillant@inria.fr>
Date: Tue, 7 Oct 2025 09:29:10 +0200
Subject: [PATCH] [PATCH] cmake: Don't link with all pinocchio

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ae7efca..0eaa778 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -135,7 +135,8 @@ else()
                               $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
   target_link_libraries(${PROJECT_NAME} INTERFACE Boost::serialization)
   if(CURVES_WITH_PINOCCHIO_SUPPORT)
-    target_link_libraries(${PROJECT_NAME} INTERFACE pinocchio::pinocchio)
+    target_link_libraries(${PROJECT_NAME}
+                          INTERFACE pinocchio::pinocchio_headers)
     target_compile_definitions(${PROJECT_NAME}
                                INTERFACE -DCURVES_WITH_PINOCCHIO_SUPPORT)
   endif(CURVES_WITH_PINOCCHIO_SUPPORT)
-- 
2.51.0

