From 008b14a19d6979a6b8f455ff3dcecae99c93d694 Mon Sep 17 00:00:00 2001
From: ManifoldFR <wilson.jallet@polytechnique.org>
Date: Thu, 16 Apr 2026 00:08:03 +0200
Subject: [PATCH] Use pinocchio_headers target

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1c320c4..3d13a90 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -147,7 +147,7 @@ else()
     )
     target_link_libraries(${PROJECT_NAME} INTERFACE Boost::serialization)
     if(CURVES_WITH_PINOCCHIO_SUPPORT)
-        target_link_libraries(${PROJECT_NAME} INTERFACE pinocchio::pinocchio)
+        target_link_libraries(${PROJECT_NAME} INTERFACE pinocchio::pinocchio_headers)
         target_compile_definitions(
             ${PROJECT_NAME}
             INTERFACE -DCURVES_WITH_PINOCCHIO_SUPPORT
-- 
2.53.0

